pkg/maps/policymap/policymap.go

```diff
@@ -235,18 +240,43 @@ func (p PolicyEntriesDump) Less(i, j int) bool {
 		p[i].Key.Identity < p[j].Key.Identity
 }
 
+// prefixMaskMap is the map of all the possible prefix length masks
+// of the port field in the policy key, corresponding to the port
+// length of that mask. The "16" prefix len implies a full mask,
+// and the default value should be "0" anyways.
+var prefixMaskMap = map[uint32]uint16{
```

Comment:
nit: The code is accurate here, but I found it confusing to call this a `mask`. I don't see it being used for any masking operations in the code. Isn't this a map from a prefix length to the highest value of the corresponding suffix?

Reply:
Renamed and updated comment to be more clear.